bciers/apps/registration/app/components/operations/registration/OperationInformationForm.tsx

```diff
@@ -34,7 +34,9 @@ const OperationInformationForm = ({
   steps,
 }: OperationInformationFormProps) => {
   const router = useRouter();
-  const [selectedOperation, setSelectedOperation] = useState("");
+  const [selectedOperation, setSelectedOperation] = useState(
+    rawFormData.operation,
+  );
```

Comment:
Because we weren't setting the initial value to the operation if it existed, the onChange was constantly calling `handleSelectOperationChange`.